vuepress/docs/v6.3.2/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,562 +1,225 @@
+There are numerous assets installed as part of the Entando PDA plugin. Entando Bundles can include more or less components, depending on objectives. It is recommended that organizations develop guidelines for bundle sizing that fit the goals of their applications and teams.
 
-Decision Model and Notation is a standard published by the Object
-Management Group. It is a standard approach for describing and modeling
-repeatable decisions within organizations to ensure that decision models
-are interchangeable across organizations.
+### Prerequisites
 
-## Back End for Front End (BFF)
+- An Entando Application on any Kubernetes provider. Follow one of the [tutorials](../#operations) appropriate to your environment to install the Entando platform.
+- The ent command line tool, installed and connected to your Kubernetes instance.
+- Red Hat PAM
 
-A microservice architecture allows teams to iterate quickly and to
-develop technology to scale rapidly. The Backend for Frontend (BFF)
-architecture is a type of pattern built with microservices. The key
-component of this pattern is an application that connects the front-end
-of your application with the backend. This BFF Code Pattern will help
-you build that component according to IBM’s best practices.
-
-# PDA Micro Frontends
-
-This section provides an overview of each of the micro frontends (MFE)
-that are available as part of the Entando PDA plugin. In cases below
-where there are details specific to the PAM implementation they are
-provided for that MFE. When extending the integration layer to other
-engines or custom implementations it will be up to the integrating team
-to define behavior and datasource for each MFE.
-
-## Task List
-
-The task list MFE provides a user with a list of visible tasks that are
-either assigned to that user or that are potentially ownable by that
-user. In the default implementation the visible tasks are limited to a
-single process instance. At configuration time the application designer
-is given the option to select a set of columns that will be visible in
-the task list for that page.
-
-### PAM Implementation
-
-The default PAM implementation makes the top level task fields available
-in the task list for selection. It is possible to fetch task and process
-variables for rendering in the task list but these values are excluded
-by default due to the additional API call overhead required.
-
-## Task Details
-
-The task details MFE renders detailed information about a given task.
-The task details widget is intended to give the final user processing a
-task the information necessary to complete the task at hand. The task
-details are rendered in a read only grid. See the styling section below
-for changing and customizing the layout.
+### Installation Steps
 
-### PAM Implementation
+1. Apply the definitions for the bundle that comprises the PDA plugin. You'll need to adjust the `-n entando` option in each command to match your namespace or project.
```

Comment:
i had the same thought. as it's written, the bundle comprises the plugin because "comprises" is plural. however, the phrasing is such that this is questionable, and it's just as likely that the "s" is accidentally - comprises -> comprise. @nshaw??